cabal.project.blockio-uring

```diff
@@ -2,5 +2,5 @@ if(os(linux))
   source-repository-package
     type: git
     location: https://github.com/well-typed/blockio-uring
-    -- liburing-2.9
-    tag: c01dbf68de237a63b8b3ed9d357526fb6eeb6b1c
\ No newline at end of file
+    -- unboxed IOOp vectors
```

Comment:
I thought it was now _not_ unboxed IOOp vectors. [c7b5500](https://github.com/well-typed/blockio-uring/commit/c7b550063b5ce2c859db50b2ebf66d41a4f24844) reverts that.